libfuse/fs.go

```diff
@@ -300,7 +303,7 @@ func (r *Root) ReadDirAll(ctx context.Context) (res []fuse.Dirent, err error) {
 			Type: fuse.DT_Dir,
 			Name: PublicName,
 		},
-	}
+	}, platformRootDirs...)
```

Comment:
We shouldn't show these unless `r.platformParams.UseLocal` is set, right?  I don't think we want to affect the current behavior yet.